### DISCUSSION TITLE
Release v1.6.6 / 5.61.6

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2611] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

Improved trace log and fixed a few bugs.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

# Changelog

### Changed
- reworked trace log retrival to greately improving performance
- improved list/tree finder
- improved trace logging
- enabled drop admin rights option for compartment style boxes
-- Note: programs started unsandboxed can not have rights dropped, but UAC prompts and elevation from within the sandboxed are blocked

### Fixed
- fixed potential BSOD issue in the driver
- fixed crash in in trace log [#2599](https://github.com/sandboxie-plus/Sandboxie/issues/2599)
- fixed tray issue [2600](https://github.com/sandboxie-plus/Sandboxie/pull/2600) (okrc)
- fixed issues with shortcuts [#2601](https://github.com/sandboxie-plus/Sandboxie/pull/2601) (okrc)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.6.6'>Release v1.6.6 / 5.61.6</a>.</em>

**Comment by BilleBarrett** [comment]
I am happy that WinggetUI picks up the updates and installs them on my client's computers 
This is very cool
Sandboxie-Plus seem to be getting better with every update
Thanks to everyone
